src/components/Forhandsvisning.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useState } from "react";
+import {
+  ForhandsvisningModal,
+  ForhandsvisningModalProps,
+} from "@/components/ForhandsvisningModal";
+import { Button } from "@navikt/ds-react";
+
+const texts = {
+  forhandsvisning: "Forhåndsvisning",
+};
+
+export const Forhandsvisning = (
```

Comment:
Skilt ut Forhandsvisning-knapp og modal i egen komponent som brukes fra Avlys-skjema. Kan tas i bruk i flere dialogmøte-skjemaer etter hvert.

Reply:
Trodde vi hadde denne fra før, men det var kanskje bare modalen?

Reply:
Ja, det var kun modalen. Ble litt ryddigere med knapp og modal sammen slik at open-state også kan leve der i stedet for i AvlysSkjema.